spec/validations/translations_spec.rb

```diff
@@ -58,7 +58,7 @@
   context 'validate translation format when "package" is defined inside "app"' do
     context 'all the leaf nodes have defined "title" and "value"' do
       let(:translation_files) do
-        [double('AppFile', relative_path: 'translations/en-US.json', read: read_fixture_file('valid_en.json'))]
+        [double('AppFile', relative_path: 'translations/en-US.json', read: read_fixture_file('valid_en-US.json'))]
```

Comment:
I put them in separate files because they're a decent size and thought it was best to pull them out for readability.